blazarclient/v1/hosts.py

```diff
@@ -72,3 +72,40 @@ def reallocate(self, host_id, values):
         resp, body = self.request_manager.put(
             '/os-hosts/%s/allocation' % host_id, body=values)
         return body['allocation']
+
+    def list_capabilities(self, detail=False, sort_by=None):
+        url = '/os-hosts/properties'
+
+        if detail:
+            url += '?detail=True'
+
+        resp, body = self.request_manager.get(url)
+        resource_properties = body['resource_properties']
+
+        # Values is a reserved word in cliff so need to rename values column.
+        if detail:
+            for p in resource_properties:
+                p['capability_values'] = p['values']
+
+        if sort_by:
+            resource_properties = sorted(resource_properties,
+                                         key=lambda l: l[sort_by])
+        return resource_properties
+
+    def get_capability(self, capability_name):
+        resource_property = [
+            x for x in self.list_capabilities(detail=True)
+            if x['property'] == capability_name]
+
+        if resource_property:
+            del resource_property[0]['values']
```

Comment:
What if this `del` happens in `list_capabilities`? That seems a better place for it, after it is copied to its new key of 'capability_values'.